Jenkinsfile

```diff
@@ -0,0 +1,25 @@
+#!groovy
+
+// https://github.com/feedhenry/fh-pipeline-library
+@Library('fh-pipeline-library') _
+
+node('nodejs-ubuntu') {
```

Comment:
@pmccarthy Could you change the label here to be simply `'nodejs4'`, and also add this step underneath, before anything else happens?

```groovy
step([$class: 'WsCleanup'])
```